examples/embedded-sso/client/src/App.tsx

```diff
@@ -81,7 +80,7 @@ export default function App() {
         });
         const json = await response.json();
         setEditableIframeSource(
-            `https://${companyCode}.swa.${apiEnvironment}.streem.cloud#token=${json.token}`,
+            `https://${companyCode}.swaga.${apiEnvironment}.streem.cloud#token=${json.token}`,
```

Comment:
❓ What do you think about using `/embed` for the source here? I assume that's the only thing people are trying to embed.
```suggestion
            `https://${companyCode}.swaga.${apiEnvironment}.streem.cloud/embed#token=${json.token}`,
```

Reply:
Using `/embed` makes sense to me.  Do we show the expert avatar on Embed 2?  I don't think we did in Embed 1, so you couldn't see if that part was working unless you went to the full app, but still makes more sense to use `/embed`, since that's what customers are using this for.

Reply:
Sure, I can update this link to include `/embed`. Should I update the example links we provide in the README for all of other libraries with this change then?

Reply:
Yeah, for consistency that would be great, thanks!